src/index.html

```diff
@@ -2,20 +2,66 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
-      "
-    >
+    ">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__product">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__photo"
+        >
+      </div>
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">
+          Product code. 195434
+      </p>
+
+      <ul class="stars stars--4 card__stars">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+
+        <p class="stars__review">Reviews: 5</p>
+      </ul>
```

Comment:
`stars_review` should not be a part of `stars` block (remember you doing _Stars_ task – there was no such element inside that block)

what it could look like (look – `stars` acts as a standalone one)
```
card__feedback
  stars
    stars__star
    stars__star
    ...
  card__review
```

Reply:
This is what I ask about in slack, remember? We put stars BEM block into an element of card here, just wasn't sure about that🙂